lib/virt_autotest/virtual_network_utils.pm

```diff
@@ -249,16 +253,20 @@ sub enable_libvirt_log {
 }
 
 sub ssh_setup {
-    my $default_ssh_key = (!(get_var('VIRT_AUTOTEST'))) ? "/root/.ssh/id_rsa.pub" : "/var/testvirt.net/.ssh/id_rsa.pub";
+    my $default_ssh_key = (!(get_var('VIRT_AUTOTEST'))) ? "/root/.ssh/id_rsa" : "/var/testvirt.net/.ssh/id_rsa";
+    my $dt              = DateTime->now;
+    my $comment         = "openqa-" . $dt->mdy . "-" . $dt->hms('-') . get_var('NAME');
     if (script_run("[[ -f $default_ssh_key ]]") != 0) {
```

Comment:
Perhaps we should replace `-f` with `-s`. `-s` tests for a regular file that is not empty.